Client/sdk/game/CVehicle.h

```diff
@@ -251,6 +251,7 @@ class CVehicle : public virtual CPhysical
     virtual void SetBikeWheelStatus(BYTE bWheel, BYTE bStatus) = 0;
 
     virtual bool IsWheelCollided(BYTE bWheel) = 0;
+    virtual int GetWheelFrictionState(BYTE bWheel) = 0;
```

Comment:
Nit: this is not using the editorconfig format. try this:

```suggestion
    virtual bool IsWheelCollided(BYTE bWheel) = 0;
    virtual int  GetWheelFrictionState(BYTE bWheel) = 0;
```